modules/cms/helpers/File.php

```diff
@@ -1,7 +1,7 @@
 <?php namespace Cms\Helpers;
 
-use October\Rain\Filesystem\Filesystem;
 use Config;
+use File as Filesystem;
```

Comment:
Use it as File please, everywhere else does.

Reply:
The class itself is called File already...

Reply:
So it is, turns out I'm not that great at code review at 2am. Who would have thought 🤷‍♂